integreat_cms/static/src/js/rss-feed.ts

```diff
@@ -2,25 +2,38 @@
  * This file contains all functions which are needed for the RSS feed
  */
 
-const loadRssEntries = (doc: Document, firstEntry: number, lastEntry: number): string => {
+const checkIfShouldBeIncluded = (node: any): boolean => {
+    let shouldBeIncluded = false;
+    const categories: NodeListOf<HTMLLinkElement> = node.querySelectorAll("category");
+    categories.forEach((category) => {
+        if (category.innerHTML.includes("Redaktionssystem")) {
+            shouldBeIncluded = true;
+        }
+    });
+    return shouldBeIncluded;
+};
+
+const loadRssEntries = (doc: Document, lastEntry: number): string => {
```

Comment:
I am a bit out of the loop here, why get rid of the `firstEntry`? Was it unused?

Reply:
This change was reverted now. We need it again with the refactoring :)